tool/src/js/conversions.rs

```diff
@@ -455,7 +455,7 @@ impl fmt::Display for InvocationIntoJs<'_> {
                             writeln!(f, "const {option_ptr} = {};", self.invocation.scalar())?;
                             writeln!(
                                 f,
-                                "return ({option_ptr} == 0) ? null : {};",
+                                "return ({option_ptr} == 0) ? undefined : {};",
```

Comment:
Thought: You could also do something like

```
let rv;
if ({option_ptr} == 0) rv = {};
return rv;
```

(not necessarily advocating for this; what you have is fine and nice that it doesn't need a local variable)